content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -92,6 +92,18 @@ Common forms of values include:
 
 One of the [recommended labels](/docs/concepts/overview/working-with-objects/common-labels/#labels).
 
+### kubectl.kubernetes.io/restartedAt
+
+Example: `kubectl.kubernetes.io/restartedAt: "2023-03-31T15:30:00Z"`
+
+Used on: Pod
+
+In this example, the annotation indicates that the Pod was most recently restarted at 3:30 PM UTC on March 31th, 2023.
+
+This annotation is used by kubernetes to keep track of the timestamp of the most recent container restart for a particular pod. This information can be useful for debugging and monitoring purposes.
+
+The `"kubectl.kubernetes.io/restartedAt"` annotation is automatically added to a Pod's metadata by the kubelet when the Pod is restarted. It records the time that the most recent restart of the Pod occurred, as a Unix timestamp.
```

Comment:
I don't think the kubelet adds this annotation; I think `kubectl` adds it (under specific circumstances)

Reply:
@sftim, Based on my research The `kubectl.kubernetes.io/restartedAt` annotation is added by the `kubelet` component when a pod is restarted. You can use this annotation to track and monitor the restart history of your Pods, which can be helpful for troubleshooting and identifying potential issues.

Here's an example of how you can retrieve the `restartedAt` annotation value for a Pod in Kubernetes using the `kubectl` command:

`kubectl describe pod <pod_name> | grep "restartedAt"`

Annotations:  kubectl.kubernetes.io/restartedAt: 2023-03-31T16:40:36.044Z

Hope now you can approve this PR!

Reply:
@Biswajit-Mohapatra2 : Per [this codebase entry](https://github.com/kubernetes/kubectl/blob/3874cf79897cfe1e070e592391792658c44b78d4/pkg/polymorphichelpers/objectrestarter.go#L32), it is kubectl that adds the annotation to the Pod template so that the underlying Pods get reconciled via new ReplicaSets without any downtime. Please make the necessary amendments.